spec/integration/_dev/benchmark/spec.yml

```
@@ -33,6 +33,27 @@ spec:
           type: folder
           name: deploy
           $ref: "./deploy/spec.yml"
+        - description: A folder containing resources used by one or more benchmarks.
+          type: folder
+          pattern: '^[a-z0-9]+-benchmark$'
+          additionalContents: false
+          required: false
+          contents:
+            - description: A file with the config definition for the corpus generator.
+              type: file
+              pattern: '^config.yml$'
+              contentMediaType: "application/x-yaml; require-document-dashes=true"
+              required: false
+            - description: A file with the fields definition for the corpus generator
+              type: file
+              pattern: '^fields.yml$'
+              contentMediaType: "application/x-yaml; require-document-dashes=true"
+              required: false
+            - description: A file with the template for the corpus generator
+              type: file
+              pattern: '^template.ndjson$'
+              contentMediaType: "application/json"
+              required: false
```

In principle if the names are know we don't need to use regular expressions.
```suggestion
              name: 'config.yml'
              contentMediaType: "application/x-yaml; require-document-dashes=true"
              required: false
            - description: A file with the fields definition for the corpus generator
              type: file
              name: 'fields.yml'
              contentMediaType: "application/x-yaml; require-document-dashes=true"
              required: false
            - description: A file with the template for the corpus generator
              type: file
              name: 'template.ndjson'
              contentMediaType: "application/json"
              required: false
```